### PR TITLE
refactor scaffolidng for MX2 (inverses)

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
Convo with @siwelwerd in Slack:

>  Steven Clontz
>   1 hour ago
> Hmm, I stumbled on Activity 4.2.5 today...
> 11 replies
> 
> 
> Steven Clontz
>   1 hour ago
> It's been around since 2022 edition at least: Activity 4.3.2 But I think, perhaps due to things being shuffled around, it's no longer well-motivated?
> 
> 
> Steven Clontz
>   1 hour ago
> I think the flow from 4.2.4 directly into 4.2.6, or a better activity that linked them together, would be preferable?
> 
> 
> Steven Clontz
>   1 hour ago
> I'll make a PR I think
> 
> 
> Drew Lewis
>   1 hour ago
> The original reason we added it was that students had trouble with Activity 4.2.7 .  So we put 4.2.5 in there to remind them that they knew how to do this already, and then 4.2.7 went much smoother.  It's possible it used to be the first activity of the section, which helped the flow, though my memory may be fuzzy.
> 
> 
> Drew Lewis
>   1 hour ago
> It was always a bit of a non-sequiter that required the facilitator to say something to the effect of "Hey, we're going to review this real quick because it will help us with the next thing".  But there's never been anything in the text that flags the facilitator to do that.
> 
> 
> Steven Clontz
>   1 hour ago
> We also had trouble with 4.2.7 today, even after 4.2.5. But since I blanked on why the problem was there (I was trying to clumsily connect it to the previous activity, "hey the kernel's non-trivial, so this isn't a bijection, and thus irrelevant to the section")
> 
> 
> Steven Clontz
>   1 hour ago
> Thanks for the context - I think I have an idea that I'll write out
> :+1:
> 1
> 
> 
> 
> Drew Lewis
>   1 hour ago
> TLDR is yes, rewrite it, but think very carefully before completely dropping 4.2.5 :)
> :+1:
> 1
> 
> 
> 
> Drew Lewis
>   1 hour ago
> (Note the repeated phrasing from 4.2.5 to part (a) of 4.2.7 -- this was very intentional)
> 
> 
> Steven Clontz
>   1 hour ago
> side note - Activity 4.2.4 went very well today. I'd split up similar card sort activities into multiple parts/activities in an earlier section, but I plan to make a PR reverting that change
> :+1:
> 1
> 
> 
> 
> Steven Clontz
>   1 hour ago
> it was much tougher without the artifact putting all the cards together in a single table (I ended up doing that myself on my whiteboard anyway)

File: source/03-AT/02.ptx [hunk 1]
```diff
@@ -328,6 +328,28 @@ Find the standard matrix for <m>T</m>.
     \end{array}\right]
   </me>
         </p>
+        <p>
+Since the formula for a linear transformation <m>T</m> and
```
Comment (StevenClontz) [inline source/03-AT/02.ptx:332]: Explicitly add Ax notation for applying a matrix to a vector.

File: source/04-MX/02.ptx [hunk 2]
```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
     </statement>
 </activity>
 
-<activity estimated-time='5'>
```
Comment (StevenClontz) [inline source/04-MX/02.ptx:93]: This activity computing a non-trivial kernel (which means this matrix is not invertible) is removed but the important scaffolding is implemented within the activity itself where it's needed.

File: source/04-MX/02.ptx [hunk 3]
```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
...
+Let <m>T: \IR^3 \rightarrow \IR^3</m> be the linear bijection given by the standard matrix
+<m>A=\left[\begin{array}{ccc} 2 &amp; -1 &amp; -6 \\ 2 &amp; 1 &amp; 3 \\ 1 &amp; 1 &amp; 4 \end{array}\right]</m>.
         </p>
     </introduction>
   <task>
       <p>
-  Write an augmented matrix representing the system of equations given by <m>T(\vec x)=\vec{e}_1</m>,
-  that is, <m>A\vec x=\left[\begin{array}{c}1 \\ 0 \\ 0 \end{array}\right]</m>.
-  Then solve <m>T(\vec x)=\vec{e}_1</m> to find <m>T^{-1}(\vec{e}_1)</m>.
+To find <m>\vec x = T^{-1}(\vec{e}_1)</m>, we need to find the unique
```
Comment (StevenClontz) [inline source/04-MX/02.ptx:127]: To get students started on this problem, I've added an MCQ that helps them unravel $T(\vec x)=\vec e_1$ into the appropriate linear system.

File: source/04-MX/02.ptx [hunk 4]
```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
...
+T^{-1}(\vec e_2)\hspace{1em}T^{-1}(\vec e_3)]
+</me>, the standard matrix for <m>T^{-1}</m>.
       </p>
   </task>
 </activity>
 
 <sage language="octave">
 </sage>
 
-<observation>
```
Comment (StevenClontz) [inline source/04-MX/02.ptx:164]: I removed this observation as I'd prefer students to take extra time to break down into the inverse transformations of standard basic vectors. (A quick RREF shortcut isn't needed when the inverse can just be computed directly anyway.)

File: source/04-MX/02.ptx [hunk 5]
```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
...
-</observation>
-
 
 <activity estimated-time='5'>
     <statement>
         <p>
-  Find the inverse <m>A^{-1}</m> of the matrix
-  <m>A=\left[\begin{array}{cc} 1 &amp; 3 \\ 0 &amp; -2 \end{array}\right]</m>
-  by row-reducing <m>[A\,|\,I]</m>.
+Find the inverse <m>A^{-1}</m> of the matrix
```
Comment (StevenClontz) [inline source/04-MX/02.ptx:191]: Made this more like questions asked in quizzes, emphasizing breaking down computation into "what happens to the basic stuff?"

File: source/04-MX/02.ptx [hunk 6]
```diff
@@ -199,7 +204,13 @@ by row reducing the matrix <m>[A\,|\,I]</m> at once.
 <activity estimated-time='5'>
     <statement>
         <p>
-Is the matrix <m>\left[\begin{array}{ccc} 2 &amp; 3 &amp; 1 \\ -1 &amp; -4 &amp; 2 \\ 0 &amp; -5 &amp; 5 \end{array}\right]</m> invertible? Give a reason for your answer.
+Is the matrix <m>\left[\begin{array}{ccc} 2 &amp; 3 &amp; 1 \\ -1 &amp; -4 &amp; 2 \\ 0 &amp; -5 &amp; 5 \end{array}\right]</m> invertible?
```
Comment (StevenClontz) [inline source/04-MX/02.ptx:207]: Made this an MCQ.

File: source/04-MX/02.ptx [hunk 7]
```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
...
+<m>A</m>.
         </p>
         <p>
-  Write an augmented matrix representing the system of equations given by <m>T(\vec x)=\vec{0}</m>,
-  that is, <m>A\vec x=\left[\begin{array}{c}0 \\ 0 \\ 0 \end{array}\right]</m>.
-  Then solve <m>T(\vec x)=\vec{0}</m> to find the kernel of <m>T</m>.
+By item (B) from <xref ref="MX2-card-sort"/>
+we may define an <term>inverse map</term><idx>inverse map</idx>
+<m>T^{-1} : \IR^n \rightarrow \IR^n</m> 
+that assigns <m>\vec x=T^{-1}(\vec b)</m> to be the unique vector such that
```
Comment (siwelwerd) [inline source/04-MX/02.ptx:103]: This wording is confusing.

File: source/04-MX/02.ptx (hunk 3 above)
Comment (siwelwerd) [reply source/04-MX/02.ptx:127]: I like this.

File: source/04-MX/02.ptx [hunk 8]
```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
...
 <activity estimated-time='5'>
     <statement>
         <p>
-  Find the inverse <m>A^{-1}</m> of the matrix
-  <m>A=\left[\begin{array}{cc} 1 &amp; 3 \\ 0 &amp; -2 \end{array}\right]</m>
-  by row-reducing <m>[A\,|\,I]</m>.
+Find the inverse <m>A^{-1}</m> of the matrix
+<me>A=\left[\begin{array}{cccc} 0 &amp; 0 &amp; 0 &amp; -1 \\ 1 &amp; 0 &amp; -1 &amp; -4 \\ 1 &amp; 1 &amp; 0 &amp; -4 \\ 1 &amp; -1 &amp; -1 &amp; 2 \end{array}\right]</me>
+by computing how it transforms each of the standard
+basic vectors for <m>\mathbb R^4</m>:
```
Comment (siwelwerd) [inline source/04-MX/02.ptx:194]: Typo -- basic-->basis

File: source/04-MX/02.ptx [hunk 9]
```diff
@@ -90,105 +90,110 @@ and a group that means <m>T</m> is <em>bijective</em>.
...
+<m>A^{-1}\vec e_1</m>, <m>A^{-1}\vec e_2</m>, <m>A^{-1}\vec e_3</m>,
```
Comment (siwelwerd) [inline source/04-MX/02.ptx:195]: I think I'd prefer $T^{-1}(\vec{e_1})$, etc.  Wondering that this extra notation, which hasn't really been used before will just increase the cognitive load.

File: source/03-AT/02.ptx (hunk 1 above)
Comment (siwelwerd) [reply source/03-AT/02.ptx:332]: Why is this the right spot for this?  Now we have some matrix multiplication going on before we define matrix multiplication.

Comment (siwelwerd) [reply source/03-AT/02.ptx:332]: Thinking more about this, maybe what I am suggesting is to here only introduce $A\vec{x}$, or $A\left[\begin{array}{c}x_1 \\ x_2 \\ x_3 \\ x_4\end{array}\right]$, with a note that the reason for this choice will be explained in Section 4.1.  Then add an activity at the end of 4.1 where they actually multiply the matrix $A$ by the vector $\left[\begin{array}{c}x_1 \\ x_2 \\ x_3 \\ x_4\end{array}\right]$.

Comment (StevenClontz) [reply source/03-AT/02.ptx:332]: Agreed - here we're just using $A\vec x$ as a formal notation, and I love the idea of confirming this lines up with matrix multiplication later. Will add that to this PR.

File: source/04-MX/02.ptx (hunk 9 above)
Comment (StevenClontz) [reply source/04-MX/02.ptx:195]: this stuck out to me too. will address

File: source/04-MX/02.ptx (hunk 7 above)
Comment (StevenClontz) [reply source/04-MX/02.ptx]: see [f01594e](https://github.com/TeamBasedInquiryLearning/linear-algebra/pull/302/commits/f01594e6414d3c40dc51eb26a08eba9437a4d502) and let me know if that's not a sufficient improvement

File: source/04-MX/02.ptx (hunk 9 above)
Comment (StevenClontz) [reply source/04-MX/02.ptx]: Addressed by [bfa25dc](https://github.com/TeamBasedInquiryLearning/linear-algebra/pull/302/commits/bfa25dc92ce36780aec149a6a9c6f614e2d46a41)

File: source/03-AT/02.ptx (hunk 1 above)
Comment (StevenClontz) [reply source/03-AT/02.ptx:332]: see [9d3e273](https://github.com/TeamBasedInquiryLearning/linear-algebra/pull/302/commits/9d3e2738ec99916c8402b0c47efd24295023ff03)

**Review by siwelwerd — APPROVED** [review]
